IntelliTect.TestTools.Selenate/GoogleSearch/GoogleHarness.cs

```diff
@@ -13,7 +14,7 @@ public GoogleHarness(GoogleBrowser browser)
             Browser = browser;
         }
 
-        public string URL => "https://www.google.com";
+        public static Uri URL => new Uri("https://www.google.com");
```

Comment:
Given that this isn't changing you can make it a readonly property.

Reply:
Changed to a... auto-property? Can't remember the name. Anyway, it's now URL { get;} = new...